numpy/random/mtrand.pyx

```diff
@@ -379,6 +379,10 @@ cdef class RandomState:
 
           (b - a) * random_sample() + a
 
+        .. note::
+            New code should use ``default_rng().random`` (see
+            `random-quick-start`) instead.
+
```

Comment:
Add this `note` and the `See Also` fragment to each of the exposed functions

Reply:
Makes sense, short and to the point.

Reply:
Per my comment in the `random-quick-start`, I really don't like the `default_rng().whatever()` construction. People should _not_ be making a new `Generator` every line. If it's really important that people have a replacement for the `np.random.whatever()` calls that fits on one line, make those module-level aliases against a global `Generator`.

Reply:
1. Instantiating a new `Generator` each time means more reading of `/dev/urandom`. While we don't pull a lot of data from it, it's still a system call, which is slow, and is going to block other processes trying to read it at the same time. If someone is mechanically replacing `np.random.whatever()` with `np.random.default_rng().whatever()`, that could be a lot of calls in tight loops.

2. I fear that this usage, especially when presented in this context, is going to engrain the idea that `default_rng()` returns _the_ default `Generator` instance rather than creating a new one each time since that was the mental model of `np.random.whatever()`.

Reply:
+1 on these comments. Perhaps just note the new name of the function when using the default_rng rather than suggesting a specific function call?

Perhaps something along the lines of 

New code should use the `random` method of a `default_rng()` instance (see `random-quick-start` for more detail).

Reply:
>  is going to engrain the idea that `default_rng()` returns _the_ default `Generator`

Perhaps we got the name wrong, and should call this `make_default_rng`?

At the very least, this sounds like something that needs to be drawn attention to in the docstring of `default_rng`.

Reply:
It is. https://docs.scipy.org/doc/numpy/reference/random/generator.html#numpy.random.default_rng

Reply:
I like the "Use the *** method of a default_rng() instance ..." text. Any other alternatives?

Reply:
changed